.gitignore

```diff
@@ -116,6 +120,9 @@ coverage/
 !/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
 !/dev/ci/**/Gemfile.lock
 
+# Exceptions to above rules for generated files
```

Comment:
The two `rx_dart` exceptions are there because the `build_runner` doesn't want to generate the files for some reason. The `rx_dart` package doesn't have the `build_runner` dependency, but the file won't be generated even when added (and the exception removed). Same case with the `rx_bloc_test`.

As for the `flutter_rx_bloc` exceptions, we do have the `build_runner` command in the github workflow but it also doesn't want to generate the files. 

@StanevPrime

Reply:
So basically you have proceed with a temporary workaround. Let's try to investigate and fix the issue accordingly.